demisto_sdk/commands/create_artifacts/tests/content_creator_test.py

```diff
@@ -185,3 +187,28 @@ def test_copy_file_to_artifacts(self):
         file_path = os.path.join(self._files_to_artifacts_dir, filename)
         content_creator.copy_file_to_artifacts(file_path)
         assert filecmp.cmp(file_path, os.path.join(self.content_repo, filename))
+
+    def test_content_legacy_with_no_md(self):
+        """
+        Given
+        - valid content dir
+        When
+        - copying the content folder to a content bundle(flatten files)
+        Then
+        - Ensure no md files were copied to content bundle
+        - Ensure md files were copied to packs bundle.
+        """
+        temp_bundles = mkdtemp(dir=self.valid_content_repo)
+        bundle_packs = os.path.join(temp_bundles, 'bundle_packs')
+        bundle_content = os.path.join(temp_bundles, 'bundle_content')
+        content_creator = ContentCreator(artifacts_path=temp_bundles, content_version='2.5.0',
+                                         preserve_bundles=True, no_update_commonserver=True)
+        with ChangeCWD(self.valid_content_repo):
+            content_creator.create_content()
+
+            assert filecmp.cmp(f'{self.valid_content_repo}/Packs/FeedAzureValid/ReleaseNotes/1_1_1.md',
+                               f'{bundle_packs}/FeedAzureValid/ReleaseNotes/1_1_1.md')
+            assert f'{self.valid_content_repo}/Packs/FeedAzureValid/CHANGELOG.md' not in bundle_content
```

Comment:
the assertion here is wrong, `bundle_content` is just a path of the bundle
we need to check the file does not exist in the bundle